### ISSUE TITLE
Is the version 2.5.21 update on Chrome Web Store official?

### ISSUE THREAD

**Issue by Hs-Yeah** [FelisCatus/SwitchyOmega#2007]
### SwitchyOmega version / SwitchyOmega 版本
2.5.21

### Browser version & OS version / 浏览器名称、版本及操作系统版本
Google Chrome

### Problem description / 问题描述
The extension on Google's Chrome Web Store is updated to version 2.5.21 recently but this version was never released.
As @FelisCatus is not active (neither on GitHub nor on [Twitter](https://twitter.com/_catus)) for a long time, I have doubts about the new version.

**Comment by FelisCatus** [comment]
tl;dr v2.5.21 was uploaded and published by myself.

In compliance with the updated Chrome Web Store policy, I've updated the extension manifest and removed the duplicated permissions for `http://*/*` and `https://*/*`. I've also clarified the rationale of each remaining permission, as requested by the CWS team.

The changes are only for compliance and DO NOT affect the functionality of the extension. There are no new permissions added or any code changes.

Also, sorry for the long delay in replying. I'm busy with my personal life and does not have the time or energy to maintain this project at this point. As the update was metadata-only, I did not push a new commit to GitHub in my haste either. I apologize again for the confusion.

**Comment by SJH114488406** [comment]
> tl;dr v2.5.21 was uploaded and published by myself.
> 
> In compliance with the updated Chrome Web Store policy, I've updated the extension manifest and removed the duplicated permissions for `http://*/*` and `https://*/*`. I've also clarified the rationale of each remaining permission, as requested by the CWS team.
> 
> The changes are only for compliance and DO NOT affect the functionality of the extension. There are no new permissions added or any code changes.
> 
> Also, sorry for the long delay in replying. I'm busy with my personal life and does not have the time or energy to maintain this project at this point. As the update was metadata-only, I did not push a new commit to GitHub in my haste either. I apologize again for the confusion.

Thanks for the work!
May I asked if the extension on the Microsoft Edge Store official? It was also been updated to 2.5.21 recently.

**Comment by FelisCatus** [comment]
> May I asked if the extension on the Microsoft Edge Store official? It was also been updated to 2.5.21 recently.

See https://github.com/FelisCatus/SwitchyOmega/issues/2004#issuecomment-667794533

**Comment by dryngoo** [comment]
Hello there! How to exclude a specific website from the switchyomega (whitelist ) and makes it use my local ip. for example there are sensitive websites such as paypal... is it possible to have an automatic switch from proxy I used to my local ip by excluding paypal from the omega switchy ?

**Comment by papampi** [comment]
Hi
Is there going to be an update for the new chrome requirements so we don't lose this amazing extension?

![image](https://github.com/user-attachments/assets/c2eec428-2813-46c1-a947-bc852344251c)

**Comment by AnyWAT** [comment]
https://github.com/FelisCatus/SwitchyOmega/issues/2513#issuecomment-2218665232
